docs/_extra/api-reference/hypothesis.yaml

```diff
@@ -344,6 +344,24 @@ paths:
             Alias of `uri`.
           required: false
           type: string
+        - name: wildcard_uri
+          in: query
+          description: |
+            Limit the results to annotations matching the wildcard URI. 
+            URI can be a URL (a web page address) or a URN representing another 
+            kind of resource such as DOI (Digital Object Identifier) or a 
+            PDF fingerprint.
+
+            `*` will match any character sequence (including an empty one), 
+            and a `?` will match any single character. `*`'s and `?`'s are not
+            permitted within the domain of the URI.
+
+            Escaping wildcards is not supported.
+
+            Examples of valid uris; `http://foo.com/*` `urn:x-pdf:*` `file://localhost/?bc.pdf`
+            Examples of invalid uris; `*foo.com`  `http://?bc.com`  `urn:*`  `file://*`
```

Comment:
Examples are helpful here 👍. Nitpick - I think "Examples of..." should be followed by a colon rather than semi-colon here,  since you are introducing a list of things.